core/amber/src/main/scala/edu/uci/ics/texera/web/resource/aiassistant/AiAssistantResource.scala

```diff
@@ -10,4 +13,42 @@ class AiAssistantResource {
   @RolesAllowed(Array("REGULAR", "ADMIN"))
   @Path("/isenabled")
   def isAiAssistantEnable: String = isEnabled
+
+  /**
+    * To get the type annotation suggestion from OpenAI
```

Comment:
does this only apply to OpenAI? I thought we were generating natural language prompts and the backbone LLM model was configurable. better to make this clear.